.npmrc

```diff
@@ -1 +1,2 @@
 engine-strict = true
+auto-install-peers = false
```

Comment:
Interesting, so this controls a default pnpm behavior?

Reply:
Yeah, by default pnpm will install `peerDependencies`, even if they're not present in your package.json. Turning off that behaviour is 'more correct', and helped me to manually resolve some clashes.